### PR TITLE
Create a welcome page

### LINKED ISSUE #18: Create welcome view
We should have some greeter on the index page.

### PR DESCRIPTION
**Pull request by phyrog**
Fixes #18.

screenshot from 2016-11-05 15-40-01

TODO:

- [x] Write tests

### CODE REVIEW

**Comment by tillmo** [comment]
Hm, you have omitted the specific example logics. This can lead to the impression that only DOL is supported.

**Comment by phyrog** [comment]
We could create some sort of FAQ page that we can link to (link on the `wide range of logic formats`) where we could list all supported logics. I think that would be a better place for this rather lengthy list.

**Comment by tillmo** [comment]
I fear this would not suffice. If some user thinks "this is restricted to DOL, which I do not know", then (s)he will not even click on that link.
Here is some shortened list: OWL, UML, FOL/TPTP, HOL/THF, ...

**Comment by phyrog** [comment]
How about: `Flexible --- supporting OWL, UML, FOL/TPTP, HOL/THF, and more`, where the `and more` would be linked to a FAQ page with more detail?

**Comment by tillmo** [comment]
yes, that sounds good.

File: package.json [hunk 1]
```diff
@@ -50,7 +50,6 @@
     "ember-load-initializers": "^0.5.1",
     "ember-lodash": "0.0.11",
     "ember-resolver": "^2.0.3",
-    "ember-welcome-page": "^2.0.1",
```
Comment (phyrog) [inline package.json]: This package added some unwanted stylesheets to the index page. The package only creates the default Ember index page, so we don't need it.

**Pull request merged**